pages/k8s/1.19/release-notes.md

```diff
@@ -76,6 +76,15 @@ the new [SR-IOV CNI documentation][cni-sriov].
 For a full list of the changes introduced in Kubernetes 1.19, please see the
 [upstream release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md)
 
+- MetalLB Operator
+
+MetalLB offers a software network load balancing implementation that allows for 
+LoadBalancing services in Kubernetes. This charm bundle has been made available
+in the charm store to be deployed along Charmed Kubernetes or MicroK8s. This operator
```

Comment:
...to be deployed on Charmed Kubernetes, MicroK8s, or any Kubernetes supported by Juju.